packages/ckeditor5-dev-env/lib/release-tools/utils/transformcommitforckeditor5package.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const { logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+const parserOptions = require( './parser-options' );
+const utils = require( './transform-commit-utils' );
+
+/**
+ * Parses a single commit:
+ *   - displays a log when the commit has invalid format of the message,
+ *   - filters out the commit if it should not be visible in the changelog,
+ *   - makes links to issues and user's profiles on GitHub.
+ *
+ * @param {Commit} commit
+ * @param {Boolean} [displayLog=true] Whether to display the logs.
+ * @returns {Commit}
+ */
+module.exports = function transformCommitForCkeditor5Package( commit, displayLog = true ) {
```

Comment:
In such case, this refactoring shouldn't be a separate PR because we're not able to review it. It's a WIP – a state of the code in the middle of a refactoring process.

We can merge it like it looks now but it'd be much easier to make decisions if we've seen the entire picture.